### DISCUSSION TITLE
CNY70

### DISCUSSION THREAD

**Discussion by Klaviernatum** [shanteacontrols/OpenDeck#76] [Q&A] [answered]
Hi!
Im new here...
Greetings to you all.
I'm designing a MIDI 2.0 Ready project,
in which I pretend to use CNY70 optic sensors to measure Velocity Note messages.
Basically it is an organ pedalboard with velocity sensitivity + buttons, exp. pedal, and some goodies...
Does the Open Deck read these sensors?

**Comment by paradajz** [comment, marked as answer]
Hi, it seems like that sensor has normal transistor output, so yes, you can connect it to OpenDeck.

**Reply by Klaviernatum** [reply]
Great!
Thank you for your fast reply!
